### ISSUE TITLE
Suggestion: Add a bit more explanation to (this part of) the routing guide.

### ISSUE THREAD

**Issue by leoj3n** [canjs/canjs#4546]
I know it's meant to be a general walkthrough, but at this part of the guide:

https://github.com/canjs/canjs/blame/39e09269c591131c24b440d2f73a7797e9340d77/docs/can-guides/experiment/technology/routing.md#L551-L564

I feel I would benefit from a short explanation of what is going on under the hood in layman's terms.

For instance, I don't think it's explicitly explained in this guide that `{ page: "home" }` literally means the **default value** of `page` is `"home"`, even though that is easily inferred if you squint.

Then the next part about `route.register("tasks/{taskId}", { page: "tasks" });` is what I really think needs to be explained in more technical terms what happens under the hood in CanJS.

**Comment by leoj3n** [comment]
After dissecting in the process of making this issue, I think I understand what that `route.register` call is saying. The `tasks/` part probably triggers a match for `window.location`, while the `{ page: "tasks" }` part sets the value of `page` on the `routeData` property of the `ViewModel` because this isn't done automatically just because the location matched the `tasks/*` pattern.

Would maybe also explicitly mention at that point what exactly `{taskId}` is and what it sets.

**Comment by leoj3n** [comment]
Most of the guide consists of regurgitated information, so I would inline this bit of information as well.

Just something I felt was missing, as the rest of the guide is very thorough without forcing a context switch to another page.

**Comment by frank-dspeed** [comment]
@chasenlehara should look in here and choose if that algins in @leoj3n would you offer a PR?
